systemtest/tests/integration/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -eu
+set -x
+
+. /etc/os-release 
+VERSION_MAJOR=$(echo $VERSION_ID | cut -d. -f 1)
+case "${ID}" in
+  rhel)
+    ;;
+  *)
+    curl --output-dir /etc/yum.repos.d/ -O \
+      "https://ftp.redhat.com/redhat/client-tools/client-tools-for-rhel-${VERSION_MAJOR}.repo"
+    curl -o /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release \
+      https://www.redhat.com/security/data/fd431d51.txt
+    ;;
```

Comment:
the three cases are 'rhel', 'fedora', 'centos', this logic only works for centos -> should we even try to install insights-client on fedora? If not, the test plugin needs to be able to handle insights-client not existing.

Reply:
> should we even try to install insights-client on fedora?

yes, however it is not possible at the moment, since insights-client is not available on fedora; this will require making it available on fedora & centos, eg via copr

> the test plugin needs to be able to handle insights-client not existing.

the problem is not much the test plugin, but rather rhc itself that assumes insights-client is always available...